docs/apply.md

```diff
@@ -0,0 +1,61 @@
+# How to apply
+
+1.  Read this reference guide, which will help you to apply and perform the different tests needed to evaluate your
+    solution.
+2.  If it is needed, set up the validation environment.
+3.  Execute tests according to the test workflow using the appropriate public instances or using your own deployed
+    instance.
+4.  Fill in the
+    [**FIWARE-ready** IoT Device Manual Template](https://s3.amazonaws.com/fundingbox-sites/gear%2F1517994518659-FIWARE+Ready+IoT+Device+Manual+Template+v1+%282%29.docx)
+    with the information about how to use your component within FIWARE ecosystem.
+5.  Go to `http://marketplace.fiware.org/apply`. Register, and then apply by providing all your company information.
+    Then, under the **solution/service** section, choose **FIWARE-ready IoT Device** under **category** and a proper
+    domain of application.
+6.  You might be asked to provide extra information or to have an interview with **FIWARE-ready** IoT Device reviewers
+    who might require extra information about your solution so that they can take an objective validation decision.
+7.  **FIWARE-ready** IoT Device reviewers will evaluate the documentation provided and will perform themselves the
+    tests.
+
+In the event that your application is correct and satisfies all the requirements, congratulations you are now part of
+the FIWARE Marketplace! The product will be updated to the **FIWARE-ready** IoT Devices list of commercially available
+products. As a result of the validation process, a document with the result will be provided to you.
+
+### FIWARE IoT platform
+
+Regarding IoT, the FIWARE platform offers two major sofware components:
+
+-   [FIWARE Context Broker](https://github.com/telefonicaid/fiware-orion): It is the main access point for developers
+    using the [NGSI](https://fiware.github.io/specifications/OpenAPI/ngsiv2) Interface. Developers can retrieve the
+    current context of an IoT device as a series of attributes of an entity. Developers may also send commands to
+    devices by updating command-related attributes, provided they have access rights for those operations.
+-   [FIWARE IoT Agents](https://github.com/Fiware/catalogue/tree/master/iot-agents). These components deal with
+    southbound communications from the FIWARE Context Broker to IoT devices and also translate northbound communications
+    from the device specific protocol into NGSI commands. This allows IoT integrators to connect devices, receive
+    measurements and send commands.
+
+![](img/01.png)
+
+Any IoT standard or proprietary protocol can be connected to FIWARE via the IoT Agent components. Currently FIWARE
+provides IoT Agents for:
+
+-   [IoT Agent for JSON](https://github.com/telefonicaid/iotagent-json) - a bridge between HTTP/MQTT messaging (with a
```

Comment:
IDEM, should we use the FIWARE GE owner's organization or the fiware organization? IMHO, we should use fiware organization...

Reply:
**NTC** - I disagree, the links have been consolidated towards using the source repos rather than our mirrors - look at the catalogue on FIWARE.org or GitHub `README` for example.